packages/block-library/src/columns/test/edit.native.js

```diff
@@ -293,6 +294,10 @@ describe( 'Columns block', () => {
 		// Get the first column
 		const firstColumnBlock = await getBlock( screen, 'Column' );
 		fireEvent.press( firstColumnBlock );
+		// Inner blocks batch store updates with microtasks.
+		// To avoid `act` warnings, we let queued microtasks to be executed.
+		// Reference: https://t.ly/b95nA
+		await act( async () => {} );
```

Comment:
I suppose this could eventually be replaced with `addBlock` if the helper retains its version of this.

Reply:
Yes, although this test case doesn't use `addBlock` but initial HTML ([reference](https://github.com/WordPress/gutenberg/pull/51303/files#diff-808bc213c23a4817b913208333ab10e7410cc791fe1fd73f89d4b1e742c10099R275-R277)).

Reply:
Ah, I misinterpreted the diff summary as being related to _adding_ an inner block. The reality is that this line relates to modifying an inner block already existing in the initial HTML. Apologies of the noise.